NClang.Tests/NClang.Tests.csproj

```diff
@@ -13,6 +13,6 @@
         <packagereference Include="NUnit3TestAdapter" Version="3.10.0" />
     </ItemGroup>
     <ItemGroup>
-      <ProjectReference Include="..\NClang\NClang.csproj" />
+        <ProjectReference Include="../NClang/NClang.csproj" />

```

Comment:
Is there any reason you needed these changes? Does this break under .NET Core on Unix-es? mono MSBuild handles backslashes as platform-native path separators.

Reply:
I known it can exclude, I feel it better coding manner for multiplatform world ;)

Reply:
I in general prefer slashes, but IIRC if you make changes to the .csproj on VSMac it will rewrite those paths to backslashes.

Reply:
Anyhow that's a cosmetic part that can be changed later. Thanks for tha changes!